app/models/role.rb

```diff
@@ -200,6 +200,18 @@ def self.permissions_with_descriptions
         description: 'Ability to add VI-SPDAT records',
         administrative: false,
       },
+      can_view_ce_assessment: {
+        description: "Access to view existing Coordinated Entry Assessments",
+        administrative: false,
+      },
+      can_edit_ce_assessment: {
+        description: "Ability to edit existing Coordinated Entry Assessments",
+        administrative: false,
+      },
+      can_submit_ce_assessment: {
```

Comment:
Does this get used?

Reply:
It's used for people who can't see existing completed assessments but can fill them out.  The equivalent is used for VI-SPDAT, but it's unclear if this will be yet.